culinary-canvas/capstone-project/backend/prisma/schema.prisma

```diff
@@ -25,6 +25,44 @@ model Users{
   ratings Rating[]
   userPreferences UserPreferences?
   mealPlanners MealPlanner[]
+  notifications Notification[]
+  subscriptions NotificationSubscription?
+  scrapedRecipes ScrapedRecipe[]
+}
+
+model Notification{
+  id Int @id @default(autoincrement())
+  userId Int
+  subject String
+  message String
+  recipes Json
+  read Boolean @default(false)
+  createdAt DateTime @default(now())
+  user Users @relation(fields: [userId], references:[id], onDelete: Cascade)
+}
+
+model NotificationSubscription{
+  id Int @id @default(autoincrement())
+  userId Int @unique
+  endpoint String
+  p256dh String
```

Comment:
What is this variable?

Reply:
So for the subscriptions every user has a p256dh and auth. These keys are used for encryption and servers use it to encrypt and decode the subscription sent.